devstack/lib/nfp

```diff
@@ -213,6 +213,8 @@ function create_nfp_gbp_resources {
 
     neutron router-gateway-clear\
  l3p_service_management
+    gbp l3policy-update --external-segment "" service_management
+
```

Comment:
This is fine for now. But we should check if one of the following are possible.
(1) Can we only do l3policy-update to clean the external segment and not clear the gateway.
(2) Create the l3policy with empty external segment.